Add support for a credential denial error (#321)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

#### Credential Request Errors {#credential-request-errors}

> For errors related to the Credential Request's payload, such as issues with `type`, `format`, `proof`, encryption parameters, or if the request is denied, use the specific error codes from this section instead of the generic `invalid_request` parameter defined in Section 3.1 of [@!RFC6750].

Reviewer: Was removing of MUST intentional?

Author: no - will fix

Author: https://github.com/openid/OpenID4VCI/pull/321/files#r1604060041

> For errors related to the Credential Request's payload, such as issues with `type`, `format`, `proof`, encryption parameters, or if the request is denied, use the specific error codes from this section instead of the generic `invalid_request` parameter defined in Section 3.1 of [@!RFC6750].

Author: ```suggestion
For errors related to the Credential Request's payload, such as issues with `type`, `format`, `proof`, encryption parameters, or if the request is denied, use the specific error codes from this section MUST be used instead of the generic `invalid_request` parameter defined in Section 3.1 of [@!RFC6750].
```

> For errors related to the Credential Request's payload, such as issues with `type`, `format`, `proof`, encryption parameters, or if the request is denied, use the specific error codes from this section MUST be used instead of the generic `invalid_request` parameter defined in Section 3.1 of [@!RFC6750].

Reviewer 2: current sentence is grammatically incorrect I think.

```suggestion
For errors related to the Credential Request's payload, such as issues with `type`, `format`, `proof`, encryption parameters, or if the request is denied, the specific error codes from this section MUST be used instead of the generic `invalid_request` parameter defined in Section 3.1 of [@!RFC6750].
```

> For errors related to the Credential Request's payload, such as issues with `type`, `format`, `proof`, encryption parameters, or if the request is denied, use the specific error codes from this section MUST be used instead of the generic `invalid_request` parameter defined in Section 3.1 of [@!RFC6750].

Reviewer 2: are the purpose of these changes clarification or do they have implication on the introduction of this new error code?

Author: I do believe there's some implications since I'm not sure how implementers have represented such errors today (before this change)

> For the errors specific to the payload of the Credential Request such as those caused by `type`, `format`, `proof`, encryption parameters in the request, or if the Credential Request is denied, the error codes values defined in this section MUST be used instead of a generic `invalid_request` parameter defined in Section 3.1 of [@!RFC6750].

Reviewer 3: ```suggestion
For errors related to the Credential Request's payload, such as issues with `type`, `format`, `proof`, encryption parameters, or if the request is denied, use the specific error codes from this section instead of the generic `invalid_request` parameter outlined in Section 3.1 of [@!RFC6750].
```
